iris/docs/version_switch.js

```diff
@@ -7,7 +7,7 @@
   var url_re = /\/iris\/docs\/(latest|(v\d+\.\d+\.?\d?))\//;
 
   var all_versions = {
-    'latest': 'latest (2.4)',
+    'latest': 'latest',
```

Comment:
I assume "latest" here redirects to RTD?

Reply:
Yes.  The latest will redirect to https://scitools.org.uk/iris/docs/latest which has an [index.html](https://github.com/SciTools/scitools.org.uk/blob/master/iris/docs/latest_readthedocs/index.html) with a redirect to read the docs (this is already present).

The change I made here is to just remove the 2.4 reference as this will be 3.1 now and will change again in future, so lets just keep it to ``latest``